resources/migrations/000_migrations.yaml

```diff
@@ -15173,6 +15173,14 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
+      id: v48.00-007
+      author: johnswanson
+      comment: Migrate migrate-click-through to a custom migration
```

Comment:
also, we need a pre-condition to only run this if the data migration has not been run before.